apps/src/music/views/SoundsPanel.tsx

```diff
@@ -220,7 +220,7 @@ const SoundsPanel: React.FunctionComponent<SoundsPanelProps> = ({
   const libraryGroupPath = library.libraryJson.path;
 
   const [selectedFolder, setSelectedFolder] = useState<SoundFolder>(
-    library.getFolderForSoundId(currentValue) || folders[0]
+    folders.find(folder => folder.id === currentValue) || folders[0]
```

Comment:
Checking my understanding - in which case would a folder's ID match `currentValue`? `currentValue` is a sound ID right?

Reply:
Thanks for catching this.  This was indeed not correct.  I've made an update which takes the "allowed sounds" into account properly when initializing the UI with an existing sound selected.

While making the update, it occurred to me that we could use the allowed sounds for all enumerations of the library. This would lead to stronger enforcement around which sounds could be used, in contrast to the current approach which only does enforcement by limiting the sounds shown in the selection UI.  That said, there might also be some risk, which is why I think it could be left for a potential follow-up.